pkg/cmd/step/verify/step_verify_preinstall.go

```diff
@@ -281,25 +281,28 @@ func (o *StepVerifyPreInstallOptions) verifyHelm(ns string) error {
 	log.Logger().Debug("Verifying Helm...")
 	// lets make sure we don't try use tiller
 	o.EnableRemoteKubeCluster()
-	_, err := o.Helm().Version(false)
+	v, err := o.Helm().Version(false)
 	if err != nil {
 		err = o.InstallHelm()
 		if err != nil {
 			return errors.Wrap(err, "failed to install Helm")
 		}
-	}
-	cfg := opts.InitHelmConfig{
-		Namespace:       ns,
-		OnlyHelmClient:  true,
-		Helm3:           false,
-		SkipTiller:      true,
-		GlobalTiller:    false,
-		TillerNamespace: "",
-		TillerRole:      "",
-	}
-	err = o.InitHelm(cfg)
-	if err != nil {
-		return errors.Wrapf(err, "initializing helm with config: %v", cfg)
+		v, err = o.Helm().Version(false)
+	}
+	if util.StartsWith(v, "v2") {
```

Comment:
Not sure whether this is correct. ` o.Helm().Version(false)` under the hood runs `helm version --short --tls` which for me for example returns `Client: &version.Version{SemVer:"v2.16.3", GitCommit:"1ee0254c86d4ed6887327dabed7aa7da29d7eb0d", GitTreeState:"clean"}`. So `util.StartsWith` won't work. 

Either one changes this check or makes _helm_cli.go_ return an actual semantic version for Helm which might be more useful.

Reply:
I only have helm3 so if you could run my code and help me fix it it would be great, this is the only thing missing for helm3 to work

Reply:
> I only have helm3 so if you could run my code and help me fix it it

Sure, as said, the main rub here is that `helm version` is returning more than a single sematic version. Let me think a bit about the best way to fix this.

What's the output of `helm version` for Helm 3? 

> is is the only thing missing for helm3 to work

This sounds intriguing. Are you saying that after this change you can use helm3 transparently and all the Jenkins X workflows still work? That would be awesome. What parts of Jenkins X are you using? How do you install it?